Docs/development/jasp-module-workflow.md

```diff
@@ -68,6 +68,8 @@ Before creating any of the other files you should now add the module folder in J
 4. Navigate to 'Advanced'
 5. Place a checkmark before 'Developer mode'
 6. Browse to the module source folder, aka a folder with the same name as your module and that contains `inst/Description.qml`.
+   
+	- When this does not work, enable libpath mode and enter your libpath into the textbox along with the name of the module you wish to load (use `.libPaths()` to identify where your packages are stored). Make sure that you have built your module in Rstudio: open the `.Rproj` located in the module source folder, go the 'Build' tab and click on 'Install'.		
```

Comment:
Technically it doesnt have to be via rstudio eh, just pure R is sufficient.
(As in ive tested this on every os without Rstudio I think)

Reply:
But maybe it is a good idea to just let them do it via rstudio, might be easier in general for people?